packages/embed/e2e/spec/functional/widget-with-parameters.spec.ts

```diff
@@ -0,0 +1,16 @@
+describe('Widget with parameters', () => {
+  before(() => {
+    cy.visit('/widget-js.html?foo=foo')
```

Comment:
Please add `bar=bar` too since it is on the list. And also 1 query param (eg.`baz=baz` that is not in the list and check its not in src

```
.invoke('attr', 'src')
.should('not.contain', 'baz')
```

Also please add the same test for `widget-html.html?...` to demonstrate it works too.

Reply:
maybe we can get rid of `widget-with-parameters.spec` and just keep all tests for widget in one single file? WDYT